packages/connect-query/src/unary-hooks.test.ts

```diff
@@ -878,8 +878,8 @@ describe('unaryHooks', () => {
         expect(result.current.get.isSuccess).toBeTruthy();
       });
 
-      expect(result.current.get.data?.count).toStrictEqual(1n);
       expect(result.current.mut.data).toStrictEqual(undefined);
+      expect(result.current.get.data?.count).toStrictEqual(0n);
```

Comment:
Did these tests need to change? Not sure i understand why they would...

Reply:
SAME.  that's why I called it out.  This is what it takes to make them pass and I'm pretty confused as to why.

Reply:
update: paul and I talked and decided to move forward.  we have other things than this to investigate going forward next week, anyway.